website/content/en/docs/building-operators/helm/tutorial.md

```diff
@@ -0,0 +1,323 @@
+
+Instead of creating your project with a boilerplate Helm chart, you can also use `--helm-chart`, `--helm-chart-repo`, and `--helm-chart-version` to use an existing chart, either from your local filesystem or a remote chart repository.
+
+If `--helm-chart` is specified, the `--group`, `--version`, and `--kind` flags become optional. If left unset, the default will be:
+
+| Flag | Value |
+| :--- | :---    |
+| domain | my.domain |
+| group | charts |
+| kind |  deduce from the specified chart |
+| version | v1alpha1 |
+
+If `--helm-chart` is a local chart archive (e.g `example-chart-1.2.0.tgz`) or directory, 
+it will be validated and unpacked or copied into the project.
+
+Otherwise, the SDK will attempt to fetch the specified helm chart from a remote repository.
+
+If a custom repository URL is not specified by `--helm-chart-repo`, the following chart reference formats are supported:
+
+- `<repoName>/<chartName>`: Fetch the helm chart named `chartName` from the helm
+                            chart repository named `repoName`, as specified in the
+                           `$HELM_HOME/repositories/repositories.yaml` file.
+                            Use [`helm repo add`](https://helm.sh/docs/helm/helm_repo_add) to configure this file.
+
+- `<url>`: Fetch the helm chart archive at the specified URL.
+
+If a custom repository URL is specified by `--helm-chart-repo`, the only supported format for `--helm-chart` is:
+
+- `<chartName>`: Fetch the helm chart named `chartName` in the helm chart repository
+                 specified by the `--helm-chart-repo` URL.
+
+If `--helm-chart-version` is not set, the SDK will fetch the latest available version of the helm chart. Otherwise, it will fetch the specified version. The option `--helm-chart-version` is not used when `--helm-chart` itself refers to a specific version, for example when it is a local path or a URL.
+
+**Note:** For more details and examples run `operator-sdk init --plugins=helm --help`.
+
+### Operator scope
+
+Read the [operator scope][operator-scope] documentation on how to run your operator as namespace-scoped vs cluster-scoped.
+
+
+## Customize the operator logic
+
+For this example the nginx-operator will execute the following
+reconciliation logic for each `Nginx` Custom Resource (CR):
+
+- Create a nginx Deployment if it doesn't exist
+- Create a nginx Service if it doesn't exist
+- Create a nginx Ingress if it is enabled and doesn't exist
+- Ensure that the Deployment, Service, and optional Ingress match the desired configuration (e.g. replica count, image, service type, etc) as specified by the `Nginx` CR
+
+### Watch the Nginx CR
+
+By default, the nginx-operator watches `Nginx` resource events as shown
+in `watches.yaml` and executes Helm releases using the specified chart:
+
+```yaml
+# Use the 'create api' subcommand to add watches to this file.
+- group: example.com
+  version: v1alpha1
+  kind: Nginx
+  chart: helm-charts/nginx
+# +kubebuilder:scaffold:watch
+```
+
```

Comment:
Bottom note about adding additional APIs, might be placed here, to be relevant.
```suggestion
**NOTE** Additional CR/CRD's can be added to the project by running, for example, the command :`operator-sdk create api --group=example --version=v1alpha1 --kind=AppService`
```

Reply:
I did not add it at the latest version. After check all again, I saw that it is at the end of the file already.